<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HiPS2FITSForm</class>
 <widget class="QWidget" name="HiPS2FITSForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HiPS2FITS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1">
      <widget class="QComboBox" name="comboSystem">
       <property name="toolTip">
        <string>Coordinate frame system to be used for the projection</string>
       </property>
       <item>
        <property name="text">
         <string>icrs</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>galactic</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Dec</string>
       </property>
       <property name="buddy">
        <cstring>lineDec</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>HiPS survey</string>
       </property>
       <property name="buddy">
        <cstring>comboHiPS</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>RA</string>
       </property>
       <property name="buddy">
        <cstring>lineRA</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineDec">
       <property name="toolTip">
        <string>Declination in decimal degrees of the center of the output image</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Width (px)</string>
       </property>
       <property name="buddy">
        <cstring>lineWidth</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QComboBox" name="comboHiPS">
       <property name="toolTip">
        <string>ID or keyword identifying the HiPS to use</string>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QComboBox" name="comboProjection">
       <property name="toolTip">
        <string>Name of the requested projection</string>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>AZP - zenithal/azimuthal perspective</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SZP - slant zenithal perspective</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TAN - tangential</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>STG - stereographic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SIN - orthographic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ARC -  zenithal/azimuthal equidistant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ZEA - zenithal/azimuthal equal area</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AIR - Airy’s projection</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CYP - cylindrical perspective</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CEA - cylindrical equal area</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CAR - plate carrée</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MER - Mercator’s projection</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SFL - Sanson-Flamsteed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PAR - parabolic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>MOL - Mollweide’s projection</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>AIT - Hammer-Aitoff</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TSC - tangential spherical cube</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CSC - COBE quadrilateralized spherical cube</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>QSC - quadrilateralized spherical cube</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HPX - HEALPix</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>XPH - HEALPix polar, aka “butterfly”</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Rotation angle (degrees)</string>
       </property>
       <property name="buddy">
        <cstring>lineRotationAngle</cstring>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Max Cut (percentile)</string>
       </property>
       <property name="buddy">
        <cstring>lineMaxCut</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineFoV">
       <property name="toolTip">
        <string>Size (FoV) of the cutout on the sky</string>
       </property>
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Projection</string>
       </property>
       <property name="buddy">
        <cstring>comboProjection</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineHeight">
       <property name="toolTip">
        <string>Height in pixels of the output image</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="comboFormat">
       <property name="toolTip">
        <string>Format of the output image</string>
       </property>
       <item>
        <property name="text">
         <string>fits</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>jpg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>png</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineRA">
       <property name="toolTip">
        <string>Right ascension in decimal degrees of the center of the output image</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineRotationAngle">
       <property name="toolTip">
        <string>Angle value (in decimal degrees) to be applied to the projection</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>CoordSys</string>
       </property>
       <property name="buddy">
        <cstring>comboSystem</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>FoV (degrees)</string>
       </property>
       <property name="buddy">
        <cstring>lineFoV</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Format</string>
       </property>
       <property name="buddy">
        <cstring>comboFormat</cstring>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Min Cut (percentile)</string>
       </property>
       <property name="buddy">
        <cstring>lineMinCut</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineWidth">
       <property name="toolTip">
        <string>Width in pixels of the output image</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Height (px)</string>
       </property>
       <property name="buddy">
        <cstring>lineHeight</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Stretch</string>
       </property>
       <property name="buddy">
        <cstring>comboStretch</cstring>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QComboBox" name="comboStretch">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Stretch function used for contrast adjustment</string>
       </property>
       <item>
        <property name="text">
         <string>linear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>power</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>sqrt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>log</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>asinh</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="lineMinCut">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Minimal value considered for contrast adjustment normalization</string>
       </property>
       <property name="text">
        <string>0.5</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="lineMaxCut">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Maximal value considered for contrast adjustment normalization</string>
       </property>
       <property name="text">
        <string>99.5</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Download</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboHiPS</tabstop>
  <tabstop>lineWidth</tabstop>
  <tabstop>lineHeight</tabstop>
  <tabstop>lineRA</tabstop>
  <tabstop>lineDec</tabstop>
  <tabstop>comboProjection</tabstop>
  <tabstop>lineFoV</tabstop>
  <tabstop>comboSystem</tabstop>
  <tabstop>lineRotationAngle</tabstop>
  <tabstop>comboFormat</tabstop>
  <tabstop>lineMinCut</tabstop>
  <tabstop>lineMaxCut</tabstop>
  <tabstop>comboStretch</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
